Build fails with certificate issue

Hi there, I am trying to build from source for my DS220+/DSM 7.0 (installing the available release gives "Invalid file format").

The command I am running is
```
sudo docker run --rm --privileged --env PACKAGE_ARCH=geminilake --env DSM_VER=7.0 -v $(pwd)/artifacts:/result_spk -v $(pwd)/tarballs:/toolkit_tarballs synobuild
```

The build is failing with the following error
```
wget https://netfilter.org/projects/libmnl/files/libmnl-1.0.4.tar.bz2
--2021-10-08 06:28:17--  https://netfilter.org/projects/libmnl/files/libmnl-1.0.4.tar.bz2
Resolving netfilter.org (netfilter.org)... 92.243.18.11, 2001:4b98:dc0:43:216:3eff:fe87:a456
Connecting to netfilter.org (netfilter.org)|92.243.18.11|:443... connected.
ERROR: cannot verify netfilter.org's certificate, issued by 'CN=R3,O=Let\'s Encrypt,C=US':
  Unable to locally verify the issuer's authority.
To connect to netfilter.org insecurely, use `--no-check-certificate'.
Makefile:26: recipe for target 'libmnl-1.0.4.tar.bz2' failed
```

I saw that you attempt to mitigate this issue by copying the certificates into the build env on L89 of `build.sh`, but it doesn't seem to be working for some reason. Any ideas why this might be the case?

--- a/build.sh
+++ b/build.sh
@@ -87,6 +87,11 @@ if [ ! -d "$build_env" ]; then
     # Ensure the installed toolchain has support for CA signed certificates.
     # Without this wget on https:// will fail
     cp /etc/ssl/certs/ca-certificates.crt "$build_env/etc/ssl/certs/"
+    
+    # Add patched version of DST Root CA X3 certificate
+    wget -O DSTRootCAX3_Extended.crt "https://crt.sh/?d=8395"
+    sed -i "s/xMDkzMDE0MDExNVow/0MDkzMDE4MTQwM1ow/g" DSTRootCAX3_Extended.crt
+    cat DSTRootCAX3_Extended.crt >> "$build_env/etc/ssl/certs/ca-certificates.crt"
 fi
 
 # Patch WireGuard to use its own included memneq implementation if architecture

